modules/exploits/linux/http/tr064_ntpserver_cmdinject.rb

```diff
@@ -104,6 +104,7 @@ def execute_command(cmd, opts)
   def check
     begin
       res = send_request_cgi({
+'method' => 'GET',
```

Comment:
I have made some changes in my local directory. How do I commit them to this branch?